staging/src/k8s.io/apiextensions-apiserver/test/integration/testserver/start.go

```diff
@@ -58,7 +59,7 @@ func DefaultServerConfig() (*extensionsapiserver.Config, error) {
 	if err := options.RecommendedOptions.SecureServing.MaybeDefaultWithSelfSignedCerts("localhost", nil, []net.IP{net.ParseIP("127.0.0.1")}); err != nil {
 		return nil, fmt.Errorf("error creating self-signed certificates: %v", err)
 	}
-	if err := options.RecommendedOptions.ApplyTo(genericConfig); err != nil {
+	if err := options.RecommendedOptions.ApplyTo(genericConfig, nil); err != nil {
```

Comment:
How does this not return a "admission depends on a scheme, it cannot be nil" error?

Reply:
Don't worry, it is protected by `options.RecommendedOptions.Admission = nil` . We don't need admission in integration test.